packages/terra-collapsible-menu-view/tests/wdio/collapsible-menu-view-spec.js

```diff
@@ -1,64 +1,53 @@
-  Terra.it.isAccessible();
 });
 
 // Only test viewports that have collapsed menu items
 Terra.describeViewports('Responsive Hidden Open', ['tiny', 'small', 'medium', 'large'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
+  it('should display responsive hidden open', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
     browser.click('[data-collapsible-menu-toggle]');
+    Terra.validates.element('default', { selector: '#root' });
   });
-
-  Terra.it.matchesScreenshot({ selector: '#root' });
-  Terra.it.isAccessible();
 });
 
 Terra.describeViewports('Collapsible Menu With Item Before Divider', ['tiny', 'small', 'medium', 'large'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
+  it('should display with item before divider', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/default-collapsible-menu-view');
     browser.click('[data-collapsible-menu-toggle]');
+    Terra.validates.element('default', { selector: '[data-terra-popup-content]' });
   });
-
-  Terra.it.matchesScreenshot({ selector: '[data-terra-popup-content]' });
-  Terra.it.isAccessible();
 });
 
 Terra.describeViewports('First hidden item visible when isIconOnly', ['medium'], () => {
-  before(() => browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/first-hidden-item-is-icon'));
-
-  Terra.it.matchesScreenshot();
-  Terra.it.isAccessible();
+  it('should display First hidden item visible when isIconOnly', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/first-hidden-item-is-icon');
+    Terra.validates.element('default');
+  });
 });
 
 Terra.describeViewports('Responsive Single Item', ['tiny', 'large'], () => {
-  before(() => {
-    browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/collapsible-menu-view-single-item');
+  it('should display responsive single item', () => {
+    browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/collapsible-menu-view-single-item');
     browser.moveToObject('#root', 0, 0);
+    Terra.validates.element('default', { selector: '#root' });
   });
 
   Terra.describeViewports('Single Item Group', ['enormous', 'large', 'huge', 'medium'], () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
+    it('should display single item group', () => {
+      browser.url('/raw/tests/terra-collapsible-menu-view/collapsible-menu-view/single-item-group');
+      Terra.validates.element('default');
     });
-
-    Terra.it.matchesScreenshot();
-    Terra.it.isAccessible();
   });
 
   Terra.describeViewports('Single Item Group Hidden Open', ['tiny', 'small'], () => {
```

Comment:
Something is a little off here. There are two `Terra.describeViewports` nested inside of another `Terra.describeViewports`. Can you investigate if this is intentional or an accident?

Reply:
looks like an unintentional change in this [PR](https://github.com/cerner/terra-framework/pull/737). Fixed 147992e